### ISSUE TITLE
Inaccessible title on plotOutput

### ISSUE THREAD

**Issue by gtritchie** [rstudio/shiny#2846]
### System details

Output of `sessionInfo()`:

```
R version 3.6.3 (2020-02-29)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.4

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] shiny_1.4.0.9002

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4.6         withr_2.1.2          packrat_0.5.0        digest_0.6.25        later_1.0.0         
 [6] mime_0.9             R6_2.4.1             jsonlite_1.6.1       xtable_1.8-4         magrittr_1.5        
[11] rlang_0.4.5          promises_1.1.0       tools_3.6.3          tinytex_0.22         rsconnect_0.8.16    
[16] httpuv_1.5.2         xfun_0.13            fastmap_1.0.1        compiler_3.6.3       htmltools_0.4.0.9003
```
### Example application *or* steps to reproduce the problem

Using the default app created by RStudio IDE 1.3.947, but specifically looking at:

```R
        # Show a plot of the generated distribution
        mainPanel(
           plotOutput("distPlot")
        )
```

<img width="1131" alt="screenshot of shiny app running" src="https://user-images.githubusercontent.com/10569626/80260805-8faf9f80-863d-11ea-8e66-853995310060.png">

### Describe the problem in detail
Load page into browser, try to determine title of the image (plot) using a screen reader. Unable to because the image representing the plot includes the title "Histogram of x" but that is not available as alt-text on the image.

### Expected
All non-cosmetic images must have text representation. I'm only focusing on the title here; the fact the plot itself is inaccessible is a whole other complex issue.

Failure to have alt-text on this image violates both WCAG 2.1 level A, and is a Section 508 MUST:

https://www.w3.org/WAI/WCAG21/quickref/#non-text-content

**Comment by hadley** [comment]
Related: #612

**Comment by schloerke** [comment]
Closing in favor of https://github.com/rstudio/shiny/issues/612
